### ISSUE TITLE
Cannot sign in, keep-test.py -> working, kim.py -> cannot sign in

### ISSUE THREAD

**Issue by 4MarkusMarkus** [djsudduth/keep-it-markdown#34]
Hi Great work!!! would love to use the tool :). 
As I have described above, keep-test.py -> working: 
``` bash
$python3 keep-test.py
Enter your Google account username: *****
Enter your Google Password: 
Test worked!! You've succesfully logged into Google Keep! Please try running Keep-it-Markdown or KIM!
``` 

kim.py -> cannot sign in, will ask me to put in username, but not ask for password, will throw error right after username is put in 
``` bash 
 python3 kim.py

Welcome to Keep it Markdown or KIM!

Enter your Google account username: ******

Username or password is incorrect (NoKeyringError('No recommended backend was available. Install a recommended 3rd party backend package; or, install the keyrings.alt package if you want to use the non-recommended backends. See https://pypi.org/project/keyring for details.'))
Could not excute KIM
```

I am using **Sign in with App Passwords** https://support.google.com/accounts/answer/185833 for sign in. 

running Python 3.8.10

**Comment by djsudduth** [comment]
@4MarkusMarkus - thx! Somehow you are missing a keyring secure storage module for the Keep token. I think you can still use kim with the -r parameter and it won’t try to save your Keep token.  “kim.py -r”

what operating system are you using?

**Comment by djsudduth** [comment]
@4MarkusMarkus did you try the 'kim.py -r' option yet?

**Comment by ichmoimeyo** [comment]
I also used to have sign in problems ...
I was able to solve it with ...

**pip3 install --upgrade keyrings.alt**

[... from [stackoverflow](https://stackoverflow.com/questions/53164278/missing-dependencies-causing-keyring-error-when-opening-spyder3-on-ubuntu18) ]

This way I just enter the pin# I chose and haven't had any problems since.

So my sequence now is ... 

pip install -r requirements.txt
pip3 install --upgrade keyrings.alt

python keep-test.py

python kim.py

... hope that helps.

**Comment by nveeser** [comment]
FWIW I am having the same issue...
Ubuntu 20.04
Python3 3.8.10

**Comment by djsudduth** [comment]
@nveeser - see the previous comment for the fix for keyring (I assume you're having the keyring issue)

**Comment by djsudduth** [comment]
This is addressed in #37  - closing for now
